templates/vue/src/components/NodeModal.vue

```diff
@@ -270,22 +276,31 @@
 
 <script>
 import Helpers from "../utils/Helpers"
-import Combobox from "./Combobox"
+import { tydeTypes } from "../utils/constants"
```

Comment:
To minimize merge conflicts with master, can we put all the tyde-related lines close to eachother if possible?